packages/web-react/src/types/pill.ts

```diff
@@ -12,10 +12,10 @@ export interface AriaPillElementTypeProps<T extends ElementType = 'span'> {
   elementType?: T | JSXElementConstructor<unknown>;
 }
 
-export interface SpiritPillProps<T extends ElementType = 'span', A = void, E = void>
+export interface SpiritPillProps<T extends ElementType = 'span', P = void>
   extends AriaPillElementTypeProps<T>,
     ChildrenProps,
     TransferProps {
   /** The color of the pill. */
-  color?: ActionColor<A> | EmotionColor<E> | PillColor;
+  color?: PillColor<P>;
```

Comment:
Can we just use `C` as `Color`? I use `P` for `Props` :-)

```suggestion
  color?: PillColor<C>;
```

Reply:
Alright :)